components/eventing-controller/reconciler/subscription/reconciler_test.go

```diff
@@ -996,20 +1127,45 @@ func createSubscriptionObjectsAndWaitForReadiness(givenSubscription *eventingv1a
 	return subscription, apiRule
 }
 
-// countRequestsForWebhooks returns how many webhook requests are sent for each given event-type
-func countRequestsForWebhooks(eventType1, eventType2 string) (int, int) {
-	count1, count2 := 0, 0
+// countRequestsWithInvalidEventType returns how many requests with not-clean event types are sent
+func countRequestsWithInvalidEventType() int {
+	count := 0
 	for _, v := range beb.Requests {
 		subscription, ok := v.(bebtypes.Subscription)
 		if ok && len(subscription.Events) > 0 {
 			for _, event := range subscription.Events {
-				if event.Type == eventType1 {
-					count1++
-				} else if event.Type == eventType2 {
-					count2++
+				if event.Type == reconcilertesting.EventTypeNotClean {
+					count++
+				}
+			}
+		}
+	}
+	return count
+}
+
+// countBebRequests returns how many requests for a given subscription are sent for each HTTP method
+func countBebRequests(subscriptionName string) (countGet int, countPost int, countDelete int) {
+	countGet, countPost, countDelete = 0, 0, 0
+	for req, v := range beb.Requests {
+		switch method := req.Method; method {
+		case "GET":
```

Comment:
please use http.MethodGet, http.MethodPost and http.MethodDelete

Reply:
Done